### PR TITLE
FAT-861 autotest C347622 is added

### PR DESCRIPTION
**Pull request by penfriend**
https://foliotest.testrail.io/index.php?/cases/view/347622

### CODE REVIEW

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/support/fragments/settings/circulation/otherSettings.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  selectPatronIdsForCheckoutScanning(optionsNames, checkoutTimeoutDuration) {
+    cy.wrap(optionsNames).each(optionName => {
+      if (optionName === 'Barcode') {
+        this.verifyCheckboxIsChecked('#barcode-checkbox', 'checkbox');
+      } else cy.do(Checkbox(optionName).click());
+    });
+    this.verifyCheckboxIsChecked('#checkoutTimeout', 'checkoutTimeout');
+
+    cy.do([
+      TextField({ id: 'checkoutTimeoutDuration' }).exists(),
```
Comment (IhorBohdan) [inline cypress/support/fragments/settings/circulation/otherSettings.js:34]: Move TextField to const

File: cypress/support/fragments/settings/circulation/otherSettings.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      TextField({ id: 'checkoutTimeoutDuration' }).fillIn(''),
+      TextField({ id: 'checkoutTimeoutDuration' }).fillIn((+checkoutTimeoutDuration + 1).toString()),
+      Button('Save').exists(),
```
Comment (IhorBohdan) [inline cypress/support/fragments/settings/circulation/otherSettings.js:37]: Move button to const

Comment (penfriend) [reply cypress/support/fragments/settings/circulation/otherSettings.js]: done.

File: cypress/support/fragments/settings/circulation/otherSettings.js (hunk 1 above)
Comment (penfriend) [reply cypress/support/fragments/settings/circulation/otherSettings.js]: done

File: cypress/support/fragments/settings/circulation/otherSettings.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import { Pane, Checkbox, Label, Form, Button, TextField } from '../../../../../interactors';
+
+export default {
+  waitLoading() {
+    cy.expect([
+      Pane('Other settings').exists(),
+      Form({ id: 'checkout-form' }).exists(),
```
[the remark this answers is not in the document]
Comment (penfriend) [reply cypress/support/fragments/settings/circulation/otherSettings.js]: done.

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**